randovania/games/dread/json_data/Cataris.txt

```diff
@@ -3087,7 +3096,7 @@ Extra - asset_id: collision_camera_038
       Any of the following:
           All of the following:
               Gravity Suit or After Cataris - Lower Lava Button
-              Speed Booster or Simple IBJ or Use Spin Boost
+              Space Jump or Speed Booster or Simple IBJ
           Space Jump and Lava Damage ≥ 500
```

Comment:
This should be tested if you can just wall jump out of the lava without gravity. if not, then this can be changed to Use Spin Boost. The Gravity Suit or Lava Button option though can be trivial cause of wall jumps.

Also, make sure if you see heat, cold, or lava damage to place a heat/cold run trick onto it. Taking these sorts of damage should not be trickless.